src/main/fc/config.c

```diff
@@ -156,7 +156,15 @@ PG_RESET_TEMPLATE(systemConfig_t, systemConfig,
 #endif
 
 #ifdef BEEPER
-PG_REGISTER(beeperConfig_t, beeperConfig, PG_BEEPER_CONFIG, 0);
+PG_REGISTER_WITH_RESET_FN(beeperConfig_t, beeperConfig, PG_BEEPER_CONFIG, 0);
```

Comment:
It was the fact that it was conditionally included that was the issue, this has been changed now.
(FWIW there are other PGs that do this, that led me to thinking it was not an issue)